hugo/layouts/partials/home/roadmap.html

```diff
@@ -2,26 +2,23 @@
     <div>
         <img class="code-pills-left" src="{{ "img/bgs/codepills-bg-3.png" | absURL }}" />
 
-        <article>
-            <div class="roadmap__photo">
-                <img src="{{ .roadmap.ceo_picture | relURL }}" alt="Eiso Kant" />
-            </div>
-
-            <div class="roadmap__content">
-                <h1>Our roadmap</h1>
-                {{ range $r := .roadmap.content }}
-                    <p>{{ $r }}</p>
-                {{ end }}
+        <h1>Our roadmap</h1>
+        {{ range $r := .roadmap.content }}
+            <p>{{ $r }}</p>
+        {{ end }}
 
-                <div class="roadmap__signature">
-                    <p class="roadmap__signature__name">Eiso Kant</p>
-                    <p>Co-Founder &amp; Ceo</p>
+        <div class="roadmap__footer">
+            <div class="roadmap__signature">
+                <img class="roadmap__signature__photo" src="{{ .roadmap.ceo_picture | relURL }}" alt="Eiso Kant" />
```

Comment:
I'd leave Eiso picture without `alt` content, or I'd explain what is that image (but I'd prefer the first one because the image is not relevant for a screen reader)